libs/shared/modal/src/lib/shared-modal.tsx

```diff
@@ -37,7 +37,7 @@ export const SharedModal = (props: SharedModalProps) => {
         aria-labelledby="transition-modal-title"
         aria-describedby="transition-modal-description"
         className="modal"
-        open={open}
+        open={open ? open : false}
```

Comment:
wrong implementation. In useState should set default value 'false' :   `const [open, setOpen] = useState<boolean(false);`
and change on loading in useEffect() if needed